packages/terra-switch/src/Switch.jsx

```diff
@@ -0,0 +1,145 @@
+import React, { useState, useRef } from 'react';
+import PropTypes from 'prop-types';
+import uniqueid from 'lodash.uniqueid';
+import classNames from 'classnames/bind';
+import * as KeyCode from 'keycode-js';
+import styles from './Switch.module.scss';
+
+const cx = classNames.bind(styles);
+
+const SWITCH_STATE = Object.freeze({
```

Comment:
I assume this is just a placeholder. So adding a comment to be resolved when translations are added.

Reply:
added [here](https://github.com/cerner/terra-core/blob/895c4fad7bb180e27e1919296ac90eeaa805dea1/packages/terra-switch/src/Switch.jsx#L9)